packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -256,6 +266,17 @@ export default compose(
 			clipboardBlock &&
 			canInsertBlockType( clipboardBlock.name, rootClientId );
 
+		// STOPSHIP (blixt): Under what conditions is the transform option
+		// enabled? I'm guessing we'll start with "if the block-type is
+		// Paragraph" for now? Is that something like 
+		// blockType === 'core/paragraph'; 
+		// Poking around in getBlockType, I'm not sure what strings get 
+		// returned from this, and I'm hesitant to hard-code a string here.
+		const isTransformEnabled = true; // 
+		const openTransformPicker = () => {
+			// STOPSHIP (blixt): Present the BlockTransformMenu here.
```

Comment:
How should I present the `BlockTransformMenu` from here? It'll be another `Picker`; I can think of a couple of ways to do it, but maybe there's another that's even better.

1) I could add a second `Picker` [here](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js#L214), and add some code to sometimes-present-it, as is done [here](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js#L175).

2) I could go out to `BlockMobileToolbar`'s [parent component and put something in there](https://github.com/WordPress/gutenberg/blob/d27e094d2bac7a5d1bdfd55598007563324e10e3/packages/block-editor/src/components/block-mobile-toolbar/index.native.js#L61) - this doesn't feel as nice, because why should that parent component be concerned with whether-or-not the BlockMobileToolbar needs to show another picker?

Is there a third option? I'm all ears - I'm new to this codebase so maybe there is another way to present this `Picker`!

Reply:
cc @chipsnyder lemme know what you think!